From f3f044a325a04912378957505d8e0e18734f643f Mon Sep 17 00:00:00 2001
From: Rameshkumar9159 <rameshsv06@gmail.com>
Date: Wed, 25 Sep 2024 14:38:39 +0530
Subject: [PATCH] Changed Hello, Universe to Hello, World

---
 git_org_hello.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/git_org_hello.c b/git_org_hello.c
index 0ba254d..f778b08 100644
--- a/git_org_hello.c
+++ b/git_org_hello.c
@@ -1,7 +1,7 @@
 #include <stdio.h>
 
 int main() {
-    printf("Hello, Universe!\n");
+    printf("Hello, World!\n");
     return 0;
 }
 
-- 
2.25.1

